api/src/play_asset_pack_manager.gd

```diff
@@ -105,23 +107,33 @@ func _initialize_plugin() -> Object:
 # -----------------------------------------------------------------------------
 # Helper functions that forward signals emitted from the plugin
 # -----------------------------------------------------------------------------
+func _forward_get_pack_states_success(result : Dictionary, signal_id : int):
+	var target_request : PlayAssetPackStateRequest = _request_tracker.lookup_request(signal_id)
+	target_request._on_get_asset_pack_state_success(result)
+	_request_tracker.unregister_request(signal_id)
+
+func _forward_get_pack_states_error(error : Dictionary, signal_id : int):
+	var target_request : PlayAssetPackStateRequest = _request_tracker.lookup_request(signal_id)
+	target_request._on_get_asset_pack_state_error(error)
+	_request_tracker.unregister_request(signal_id)
+
 func _forward_show_cellular_data_confirmation_success(result : int, signal_id : int):
-	var target_request = _request_tracker.lookup_request(signal_id)
+	var target_request : PlayCellularDataConfirmationRequest = _request_tracker.lookup_request(signal_id)
```

Comment:
Added type hints for these lookups.

Reply:
Godot editor has a static type checker, and Godot runtime have runtime type checks. If I change this ```PlayCellularDataConfirmationRequest``` other types the related test cases will fail.